app/serializers/serial_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class SerialSerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :thumbnail_url
+
+  has_many :posts
```

Comment:
I think this is unnecessary for https://github.com/bm-sms/nursepress/pull/265 .
This is association embeds unnecessary `relationships` response.
Such as:
``` json
{
  "data": [
    {
      "id": "2",
      "type": "serials",
      "attributes": {
        "title": "連載",
        "description": "hi",
        "thumbnail_url": "http://localhost:3000/uploads/serial/thumbnail/2/icon.png"
      },
      "relationships": {
        "posts": {
          "data": [
            {
              "id": "10205",
              "type": "posts"
            }
          ]
        }
      }
    }
  ],
  "links": {}
}
```